.vsts-ci.yml

```diff
@@ -130,20 +130,16 @@ extends:
               matrix:
                 ARM64:
                   _RID: osx-arm64
-                  _BuildConfig: Release
-                  _SignType: real
                   _DotNetPublishToBlobFeed: false
                 X64:
                   _RID: osx-x64
-                  _BuildConfig: Release
-                  _SignType: real
```

Comment:
We don't have to specify this anymore?

Reply:
No need on adding it as a variable if it doesn't change in the matrix strategy. Better just adding it directly when calling the script